Nodejs/Product/Nodejs/Snippets/1033/Nodejs/http_get.snippet

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>http_get</Title>
+      <Description>Code Snippet for handling an HTTP GET request with Express</Description>
+      <Shortcut>http_get</Shortcut>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>Hello World</ID>
+          <Default>Hello World</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[app.get('/', function (request, response) {

```

Comment:
the routes in this and similar snippets should be editable as well because people typically want to specify more than the root page, so better to use ```'$route$'``` than ```'/'```